runtime/lib/ttnn/operations/clamp/clamp.cpp

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "clamp.h"
+#include "tt/runtime/detail/logger.h"
+#include "tt/runtime/detail/ttnn.h"
+#include "tt/runtime/ttnn/operations/utils.h"
+#include "ttmlir/Target/TTNN/program_generated.h"
+#include "ttnn/operations/eltwise/unary/unary_composite.hpp"
+
+namespace tt::runtime::ttnn::operations::clamp {
+void run(const ::tt::target::ttnn::ClampOp *op, ProgramContext &context) {
+  ProgramTensorPool &tensorPool = context.getTensorPool();
+  const ::ttnn::Tensor &input = tensorPool.at(op->in()->global_id());
+  const float minValue = op->min();
+  const float maxValue = op->max();
+  ::tt::tt_metal::MemoryConfig memoryConfig =
+      utils::createMemoryConfig(op->out());
+
+  ::ttnn::Tensor out = ::ttnn::clamp(input, minValue, maxValue, memoryConfig);
```

Comment:
In TTNN clamp and clip are modelled as unary operations, I think we should match that, so unary.cpp might be a more suitable location for this. Now that I think of it, we might want to separate unary_composite from unary.cpp (I see there are already a couple unary composite ops added to unary.cpp), but for now you can add it to unary.cpp.

Reply:
TTNN clamp and clip are bit different than other composite unary ops. These two ops has three inputs; 1) input tensor, 2) min value as float, and 3) max value as float. So `unary.cpp::runEltwiseUnaryCompositeOP` can not handle these two ops due to additional float inputs unless we handle these additional cases (I don't prefer this approach as this will make code more complex). I can create a new function to handle clamp/clip in unary.cpp or move the `clamp` directory to `operations/eltwise/`. What do you think is a better way?

Reply:
This is something we're missing, to account for eltwise ops with extra parameters. I posted a PR #1121 to add support for this.

Reply:
Thanks, I'll update mine on top of your PR.

Reply:
@mmanzoorTT FYI this is merged to main